README.md

```diff
@@ -14,100 +23,160 @@ For further ansible documentation see: (https://docs.ansible.com/ansible/2.4/vau
+### **1. Build Images**
 
-1. Build the images and load with fixture data
+Before we can run the application locally, we need to build the docker images and load the fixtures:
 
 ```sh
 make docker-bootstrap
 ```
+*Known Issue: If the swagger/documentation container fails to build or start, edit the makefile locally to remove `documentation` from the `docker-bootstrap` command line.
 
-2. Start the containers
+After that has completed successfully, we can start the containers:
 
 ```sh
 docker-compose up
 ```
 
-## Test
+### **2. Get a token**
+
+Once the containers are running, you will need to generate a set of credentials for an ACO so that you can get a token. The loaded fixtures will include some ACOs that have beneficiaries attributed to them already. The ACOs loaded in the previous step can be found in the database.
+
+```sh
+ACO_CMS_ID=<> make credentials
+```
+
+This will generate a client ID and secret that can be used to acquire a token from the SSAS App:
+
+```sh
+curl --location --request POST 'http://localhost:3003/token' \
+--header 'Accept: application/json' \
+--user '<clientid:secret>'
+```
+
+### **3. Make a request**
+
+After we successfully retrieve a token, we can make a request to any of the available endpoints. The PostMan collections under `test/postman_test/...` can be imported into postman directly and used to make requests, or you can use any tool like `curl`.
+
+#
+
+## **Run Tests**
 
-Run tests and produce test metrics.
-The items identified above in the `Build/Start` section are prerequisites to running tests.
+**Prerequisite**: Before running the tests and produce test metrics, you must complete the `Build Images` step from `Start the API` section. 
 
-   *Note: If this is the first time running the tests follow instructions in the 'Running unit tests locally' section of this README. Then run:
+### **1. Seed The Database**
+Spin up the Postgres container & run migrations:
    ```sh
-   make load-fixtures
+   $ make unit-test-db
    ```
+### **2. Source Environment Variables**
+Source the required environment variables from the `./.vscode/settings.json` (under go.testEnvVars) and `./shared_files/decrypted/local.env`.
+
+   **NOTE:** Since we're connecting to Postgres externally, we need to use the local host/port instead.
+
+   For vscode users, these variables are already by the workspace settings file (`./.vscode/settings.json`)
+
+*Note: If this is the first time running the tests follow instructions in the 'Running unit tests locally' section of this README. Then run:
+```sh
+make load-fixtures
+```
 
 In order to keep the test feedback loop optimized, the following items must be handled by the caller (and are not handled by the test targets):
 
 - Ensuring the compose stack is up and running
 - Ensuring the database has been seeded
 - Managing images/containers (if Dockerfile changes have occurred, an image rebuild is required and won't occur as part of the test targets)
```

Comment:
I think these steps are only for running tests through VSCode / using `go` directly (i.e. not using make unit-test). Could we clarify that here somehow? 🤔 I've generally run `make unit-test` directly which automatically runs `make-unit-test-db` and such without me having to do anything 😄 

Thanks!

Reply:
nice, I didn't catch that previously! I'll update that as well.